client/src/pages/CreateWorkout/CreateWorkout.jsx

```diff
@@ -225,13 +225,7 @@ const CreateWorkout = () => {
         <div className="flex w-full grow flex-col items-center gap-3">
           {" "}
           {exerciseCards}
-          {showForm ? (
-            <ExerciseForm
-              exerciseFromParent={exercise}
-              handleExerciseAdded={handleExerciseAdded}
-              handleShowForm={handleShowForm}
-            />
-          ) : (
+          {showForm || (
```

Comment:
hmm curious why the change from ternary to showFrom || and showForm &&. I actually think the ternary logic is a lot clearer what is going to show when!